config/clusters/2i2c/neurohackademy.values.yaml

```diff
@@ -0,0 +1,101 @@
+dex:
+  enabled: true
+  hubHostName: neurohackademy.2i2c.cloud
+
+staticWebsite:
+  enabled: true
+  source:
+    git:
+      repo: https://github.com/neurohackademy/zoom2022
+      branch: master
+  ingress:
+    host: neurohackademy.2i2c.cloud
+    path: /zoom
+  githubAuth:
+    enabled: true
+
+jupyterhub:
+  custom:
+    2i2c:
+      add_staff_user_ids_to_admin_users: true
+      add_staff_user_ids_of_type: "github"
+    homepage:
+      templateVars:
+        org:
+          name: NeuroHackademy
+          url: https://neurohackademy.org/
+          logo_url: https://user-images.githubusercontent.com/118582/178122521-aa5b83e7-f898-415a-89ff-dc288f599b4f.png
+        designed_by:
+          name: 2i2c
+          url: https://2i2c.org
+        operated_by:
+          name: 2i2c
+          url: https://2i2c.org
+        funded_by:
+          name: The National Institutes of Health grant 2R25MH112480-06
+          url: https://reporter.nih.gov/search/ydTvTwXxk0yd6eGdRznbLQ/project-details/10409452
+  singleuser:
+    # User image: https://quay.io/repository/arokem/nh-jhub-2022?tab=tags
+    image:
+      name: quay.io/arokem/nh-jhub-2022
+      tag: "3d441bdb82f6"
+    # Uncomment this section after resolution of PR https://github.com/2i2c-org/infrastructure/pull/2758
+    # nodeSelector:
+    #   2i2c.org/community: neurohackademy
+    cpu:
+      guarantee: 0.5
+      # We're on n2-highmem-16 machines
+      limit: 14
+    memory:
+      guarantee: 4G
+      limit: 8G
```

Comment:
Hmmmm, we will fit up to 16/0.5 = 32 users or 128/4 = 32 users now, while before we would fit a few less because of lower memory capacity. Yeah I think this is more suitable now than before!

Reply:
Unfortunately we have no n2 CPU quota, so I've moved this back to n1 now.